lib/gooddata/bricks/middleware/gooddata_middleware.rb

```diff
@@ -24,6 +24,10 @@ def call(params)
         # Set logger
         logger = params['GDC_LOGGER']
         GoodData.logger = logger
+        splunk_logger = params['GDC_SPLUNK_LOGGER']
```

Comment:
Refers to 'params' more than self (maybe move it to another class?) (FeatureEnvy)